### PR TITLE
Sanitizing token.

### PR DESCRIPTION
**Pull request by AlvaroAguilera**
**Description of the Pull Request (PR):**

Small token sanitation line to fix the issue with participants copying tokens with added zero-width spaces.

**This fixes or addresses the following GitHub issues:**

- Ref: #141 

**Checkoff for all PRs:**

 - [x] I have read the [Guidelines for Contributing](https://github.com/expfactory/expfactory/blob/master/.github/CONTRIBUTING.md) and this PR conforms to the stated requirements.
 - [ ] I have updated the [CHANGELOG](https://github.com/expfactory/expfactory/blob/master/CHANGELOG.md)

Attn: @AlvaroAguilera

### CODE REVIEW

File: expfactory/views/headless.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def login_get():
     if not subid:
         token = request.args.get("token")
         if token:
-
+            token = form.token.data.encode('ascii', errors = 'ignore').decode().strip()
```
Comment (vsoch) [inline expfactory/views/headless.py:65]: This isn't technically correct - note that above we are getting the token from request.args, and then below you are again getting it (but this time from form data). I think you would just want to apply that encode to the token you've already gotten. Also look below here to see there are two cases where we retrieve the token, one from the request and one from form data, and you should test if you need to encode both.

Comment (AlvaroAguilera) [reply expfactory/views/headless.py:65]: How about putting the line inside the function validate_token()?

Comment (vsoch) [reply expfactory/views/headless.py:65]: There are actual multiple instances of that function, I think 3 or 4 places to add it, vs. adding to the original grab of the token (N=2). So I'm thinking we want it after retrieving the argument.

Comment (vsoch) [reply expfactory/views/headless.py:65]: You need to keep the same methods to retrieve the token (request.args.get) and then in the "if token" statement further do token = token.encode...

**Comment by vsoch** [comment]
@AlvaroAguilera I had requested some changes here - any updates?

**Comment by AlvaroAguilera** [comment]
I'm in debt, I know. Hope to find time for this soon.

**Pull request closed without merging**